heimdall.application/pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <artifactId>heimdall</artifactId>
-        <groupId>com.miq.automation</groupId>
+        <groupId>com.miqdigital.automation</groupId>
         <version>1.0-SNAPSHOT</version>
```

Comment:
Can we change the snapshot to a release version of 1.0?

Reply:
I will update it.